pom.xml

```diff
@@ -93,6 +93,7 @@
 
   <modules>
     <module>common</module>
+	<module>format</module>
```

Comment:
This i have added to run all the testcases, I will remove after fixing all the failures